tailwind.config.ts

```diff
@@ -1,350 +1,36 @@
-        spin: 'spin 1.5s cubic-bezier(.72,.18,.3,.99) infinite',
-        blinkingText: 'blinkingText 0.8s ease-in-out infinite alternate',
-        shake: 'shake ease-in 5s infinite',
-        appear: 'appear 1000ms normal ease forwards',
-      },
-      keyframes: {
-        slideUp: {
-          from: { opacity: '0', transform: 'translateY(10px)' },
-          to: { opacity: '1', transform: 'translateY(0)' },
-        },
-        slideDown: {
-          from: { opacity: '0', transform: 'translateY(-10px)' },
-          to: { opacity: '1', transform: 'translateY(0)' },
-        },
-        spin: {
-          '0%': { transform: 'rotate(0deg)' },
-          '100%': { transform: 'rotate(360deg)' },
-        },
-        blinkingText: {
-          from: {
-            textShadow:
-              '0 0 2px #fff, 0 0 3px #dfaa7e, 0 0 4px #dfaa7e, 0 0 5px #dfaa7e, 0 0 6px #dfaa7e, 0 0 7px #dfaa7e, 0 0 8px #dfaa7e;',
-          },
-          to: {
-            textShadow:
-              '#FFF 0px 0px 1px, #FFF 0px 0px 2px, #FFF 0px 0px 3px, #dfaa7e 0px 0px 2px, #dfaa7e 0px 0px 3px, #dfaa7e 0px 0px 4px, #dfaa7e 0px 0px 20px, #dfaa7e 0px 0px 10px;',
-          },
-        },
-        shake: {
-          '0%': { left: '0' },
-          '1%': { left: '-3px' },
-          '2%': { left: '5px' },
-          '3%': { left: '-8px' },
-          '4%': { left: '8px' },
-          '5%': { left: '-5px' },
-          '6%': { left: '3px' },
-          '7%': { left: '0' },
-        },
-        appear: {
-          '0%': { opacity: '0' },
-          '100%': { opacity: '1' },
-        },
-      },
-      lineClamp: {
-        // line clamp's default stops at 6 lines! but you can add even more:
-        10: '10',
-        12: '12',
-      },
-      colors: {
-        default: '#2B2B2B',
-        background: '#F4EDDE',
-        secondary: '#3D3D3D',
-        primary: '#488F66',
-        accent: '#EDF5FF',
-        error: '#EDF5FF',
-        success: '#26DF73',
-        dark: {
-          default: '#ECECEC',
-          background: '#11151F',
-          secondary: '#3D3D3D',
-          primary: '#6FCF97',
-          accent: '#EDF5FF',
-          error: '#EDF5FF',
-          success: '#26DF73',
-          card: '#212533',
-        },
-      },
+      spacing: { ...spacing },
+      width: { ...width },
+      minWidth: { ...minWidth },
+      maxWidth: { ...maxWidth },
+      height: { ...height },
+      minHeight: {...minHeight},
+      maxHeight: { ...maxHeight },
+      borderWidth: { ...borderWidth },
+      borderRadius: { ...borderRadius },
+      boxShadow: {...boxShadow},
+      saturate: { ...saturate },
+      contrast: {...contrast},
+      invert: {...invert},
+      translate: { ...translate },
+      scale: { ...scale },
+      backgroundImage: { ...backgroundImage},
+      rotate: {...rotate},
+      animation: { ...animation },
+      keyframes: { ...keyframes },
+      lineClamp: { ...lineClamp },
+      colors: {...colorsDefine?.colors},
```

Comment:
Can you check if in that way it's working ?
```suggestion
      spacing,
      width,
      minWidth,
      maxWidth,
      ...
```